# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../hdparm/cross-no-strip.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- hdparm-7.6/Makefile.vanilla	2007-08-10 17:47:26.000000000 +0200
+++ hdparm-7.6/Makefile	2007-08-10 17:47:32.000000000 +0200
@@ -28,7 +28,6 @@
 
 hdparm: hdparm.h $(OBJS)
 	$(CC) $(LDFLAGS) -o hdparm $(OBJS)
-	strip hdparm
  
 install: all hdparm.8
 	if [ ! -z $(DESTDIR) ]; then $(INSTALL_DIR) $(DESTDIR) ; fi
